dataland-e2etests/build.gradle.kts

```diff
@@ -71,6 +72,28 @@ tasks.register("generateBackendClient", org.openapitools.generator.gradle.plugin
     )
 }
 
+tasks.register("generateQaServiceClient", org.openapitools.generator.gradle.plugin.tasks.GenerateTask::class) {
+    val qaServiceClientDestinationPackage = "org.dataland.datalandqaservice.openApiClient"
+    input = project.file("${project.rootDir}/dataland-qa-service/qaServiceOpenApi.json").path
+    outputDir.set("$buildDir/clients/qa-service")
+    packageName.set(qaServiceClientDestinationPackage)
+    modelPackage.set("$qaServiceClientDestinationPackage.model")
+    apiPackage.set("$qaServiceClientDestinationPackage.api")
+    generatorName.set("kotlin")
+
+    additionalProperties.set(
+        mapOf(
+            "removeEnumValuePrefix" to false,
+        ),
+    )
+    configOptions.set(
+        mapOf(
+            "dateLibrary" to "java17",
+            "useTags" to "true",
+        ),
+    )
+}
+

```

Comment:
This is boilerplate in large parts.
We could create a method like this:
`fun createOpenApiGenerateTask(name: String, inputPath: String, outputDir: String, ...): GenerateTask {}`
and reuse it everytime we want to build a client.

Especially because most of the configs are the same across all client-generation-tasks.

What do you think?

Reply:
Nothing was done. The same applies as for the template for a spring boot service - the team does not think that getting rid of this boilerplate code would safe sufficient development time compared to the added complexity.